### DISCUSSION TITLE
Hi, Everybody

### DISCUSSION THREAD

**Discussion by minime89-maker** [abhisheknaiidu/awesome-github-profile-readme#664] [Show and tell]
My name is Minja, and if you have free time, consider to add my Profile to your Awesome Collection ✌️ 
🦄  https://github.com/minime89-maker

**Comment by goncharov-vlad** [comment]
Wow! Relly nice) 
Add and my too, but more simply
https://github.com/goncharov-vlad

**Reply by g0621** [reply]
checkout mine too 🤣 just dropped little bit of everything

[https://github.com/g0621/g0621](https://github.com/g0621/g0621)

**Comment by moedayraki** [comment]
Is this repo stilled maintained? 😇

**Reply by samuel-ma** [reply]
@moedayraki----I know right 
There's like 300+ pull requests pilled up and close to no mention from any of the "Maintainers"

**Reply by moedayraki** [reply]
@abhisheknaiidu Do you need help maintaining this repo?😇

**Comment by Villex-code** [comment]
Hello man, I am looking to get Galaxy brain achievement, would appreciate if you could select this answer ! Thank you man !
